assets/js/components/utils/index.js

```diff
@@ -177,3 +177,22 @@ export const getTerms = ( attribute ) => {
 		),
 	} );
 };
+
+/**
+ * Given a page object and an array of page, format the title.
+ *
+ * @param  {Object} page           Page object.
+ * @param  {Object} page.title     Page title object.
+ * @param  {string} page.title.raw Page title.
+ * @param  {string} page.slug      Page slug.
+ * @param  {Array}  pages          Array of all pages.
+ * @return {string}                Formatted page title to display.
+ */
+export const formatTitle = ( page, pages ) => {
+	if ( ! page.title.raw ) {
+		return page.slug;
+	}
+	const hasSimilar =
```

Comment:
From what I'm seeing, this isn't checking for "similar" results but duplicates..maybe this should be called `unique`, and is true when no duplicates exist?

Reply:
Nit: instead of `unique`, what about `isUnique`? I think it makes it easier to understand the function returns a boolean.